index.html

```diff
@@ -6,15 +6,15 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-     <!-- Global site tag (gtag.js) - Google Analytics -->
-     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-208629130-1"></script>
-     <script>
-         window.dataLayer = window.dataLayer || [];
-         function gtag() { window.dataLayer.push(arguments); }
-         gtag('js', new Date());
- 
-         gtag('config', 'UA-208629130-1');
-     </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-208629130-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag() { window.dataLayer.push(arguments); }
+        gtag('js', new Date());
+
+        gtag('config', 'UA-208629130-1');
+    </script>
 
```

Comment:
- [ ] On line 26, change the title page to a more meaningful name such  Okpara's portfolio